main.go

```diff
@@ -197,7 +187,7 @@ func restConfigForAPIExport(ctx context.Context, cfg *rest.Config, apiExportName
 			if !ok {
```

Comment:
Also if the event is an error type

Reply:
Actually, sorry - let's just use the `RetryWatcher`

Reply:
Exactly what I started doing since your last comment 🙂. Keep you posted!